src/stdatamodels/fits_support.py

```diff
@@ -897,11 +907,88 @@ def get_short_doc(schema):
     return description.partition('\n')[0]
 
 
-def ensure_ascii(s):
+def _ensure_ascii(s):
     # TODO: This function seems to only ever receive
     # string input.  Also it's not checking that the
     # characters in the string fall within the valid
     # range for FITS headers.
     if isinstance(s, bytes):
         s = s.decode('ascii')
     return s
+
+
+##############################################################################
+# AsdfInFits 'replacements'
+# as AsdfInFits is moving out of ASDF we will provide some public functions
+# to aid in the migration to stdatamodels. Most of the usage of AsdfInFits
+# appears to be covered by 3 different 'modes':
+#
+# Write:
+#     # note the use of a potentially non-empty hdulist
+#     ff = asdf.fits_embed.AsdfInFits(hdulist, tree)
+#     ff.write_to(filename, ...)
+#
+# Read:
+#     ff = asdf.open(filename)  # where filename is a fits file
+#       -- or explicity --
+#     ff = asdf.fits_embed.AsdfInFits.open(filename)
+#
+# Read in context:
+#     with asdf.open(filename) as af:
+#       -- or explicitly --
+#     with asdf.fits_embed.AsdfInFits.open(filename) as af:
```

Comment:
If this part of the module needs this header, then it should be its own module with this as the module's docstring.

Reply:
These were leftover comments from my own notes. I've removed them in favor of including something similar in the actual docs.